Create new endpoints for querying activities, faculties and skills lists

filipjurcak opened the pull request:
Closes #19 

In this PR, we expose 3 APIs that frontend can call, specifically: `/skills`, `/activities`, `/faculties`. All of these return lists with values corresponding to that endpoint, e.g. result of calling `/faculties` endpoint is 
```c
[
    {
        "attribute_id": 1,
        "id": 1,
        "value": "Lekárska fakulta (LF UK)"
    },
    {
        "attribute_id": 1,
        "id": 2,
        "value": "Právnická fakulta (PraF UK)"
    },
    // and so on
]
```

filipjurcak committed:
Create new endpoints for querying activities, faculties and skills lists

filipjurcak commented:
@Saamoot just briefly check please

Saamoot commented:
In file: `models.py` on lines: [193](https://github.com/uk-kona/univerzita-kona-api/blob/637571722c4ce41331d48db204ad20aadb2180f4/app/models.py#L193),  [182](https://github.com/uk-kona/univerzita-kona-api/blob/637571722c4ce41331d48db204ad20aadb2180f4/app/models.py#L182),  [163](https://github.com/uk-kona/univerzita-kona-api/blob/637571722c4ce41331d48db204ad20aadb2180f4/app/models.py#L163), [152](https://github.com/uk-kona/univerzita-kona-api/blob/637571722c4ce41331d48db204ad20aadb2180f4/app/models.py#L152) values are missing closing brackets `]` in return value. 

Check file for similar issues.

Otherwise ok.

filipjurcak committed:
Add missing square brackets to model debug message

filipjurcak commented:
> In file: `models.py` on lines: [193](https://github.com/uk-kona/univerzita-kona-api/blob/637571722c4ce41331d48db204ad20aadb2180f4/app/models.py#L193), [182](https://github.com/uk-kona/univerzita-kona-api/blob/637571722c4ce41331d48db204ad20aadb2180f4/app/models.py#L182), [163](https://github.com/uk-kona/univerzita-kona-api/blob/637571722c4ce41331d48db204ad20aadb2180f4/app/models.py#L163), [152](https://github.com/uk-kona/univerzita-kona-api/blob/637571722c4ce41331d48db204ad20aadb2180f4/app/models.py#L152) values are missing closing brackets `]` in return value.
> 
> Check file for similar issues.
> 
> Otherwise ok.

Good callout, fixed now.

I've also created a [wiki](https://github.com/uk-kona/univerzita-kona-api/wiki/API-doc) for all endpoints that we currently expose, and will update it after we implement more.

As I think I've addressed all the comments, I'll merge this one.
